File: Java/src/test/java/com/gildedrose/GoldenMasterGenerator.java
package com.gildedrose;

import java.io.*;

public class GoldenMasterGenerator {

    public static void main(String[] args) throws IOException {
        try (FileOutputStream fileOutputStream = new FileOutputStream("src/test/resources/com/gildedrose/golden-master.txt", true);
             BufferedOutputStream bufferedOutputStream = new BufferedOutputStream(fileOutputStream);
             PrintStream printStream = new PrintStream(bufferedOutputStream)) {
             System.setOut(printStream);
             TexttestFixture.main(new String[]{"50"});
        }
    }

}

File: scala/src/test/scala/com/gildedrose/GoldenMasterGenerator.scala
package com.gildedrose

import java.nio.charset.StandardCharsets
import java.nio.file.{Files, Paths}

object GoldenMasterGenerator {

  def main(args: Array[String]): Unit = {
    Files.write(Paths.get("src/test/resources/com/gildedrose/golden-master.txt"), new GildedRoseExecutor().execute(50).getBytes(StandardCharsets.UTF_8))
  }

}
